src/Adapter/Order/QueryHandler/GetOrderForViewingHandler.php

```diff
@@ -425,7 +421,9 @@ private function getOrderProducts(Order $order): OrderProductsForViewing
                 $product['unit_price_tax_excl'] :
                 $product['unit_price_tax_incl']
             ;
-            $totalPrice = Tools::ps_round($unitPrice, 2) * ($product['product_quantity'] - $product['customizationQuantityTotal']);
+
+            $totalPrice = Tools::ps_round($unitPrice, 2) *
```

Comment:
I have high doubts about ALL the Tools::ps_round use,  it only takes away some precision along the way, the only places where we should keep them is at the final rendering of API returns But not in a query return since we don't know how it will be used after (ideally it should be a Number)
Besides, assuming we keep those ugly Tools::ps_round, the precision should not be a hard coded 2 but rather it should come from `ComputingPrecision`

Reply:
Removed